src/Microsoft.TestPlatform.PlatformAbstractions/Interfaces/System/IProcessHelper.cs

```diff
@@ -90,7 +90,7 @@ public interface IProcessHelper
         /// <param name="callbackAction">
         /// Callback on process exit.
         /// </param>
-        void SetExitCallback(int processId, Action callbackAction);
+        void SetExitCallback(int processId, Action<object> callbackAction);
```

Comment:
>object [](start = 51, length = 6)

explicitly as Process or have an eventArgs..

Reply:
Since Process API's are not available in UWP, using it explicitly here would mean we cannot built uwp platform abstraction.